common/script/script-api/src/main/java/org/thingsboard/script/api/js/NashornJsInvokeService.java

```diff
@@ -116,7 +116,6 @@ public void init() {
             sandbox.setExecutor(monitorExecutorService);
             sandbox.setMaxCPUTime(maxCpuTime);
             sandbox.setMaxMemory(maxMemory);
-            sandbox.allowNoBraces(false);
```

Comment:
Due to https://github.com/javadelight/delight-nashorn-sandbox/compare/0.4.5...0.5.0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R114